# Generated by Django 3.1.2 on 2021-05-29 14:05

from django.db import migrations, models
import django.db.models.deletion
import django_jalali.db.models


class Migration(migrations.Migration):

    dependencies = [
        ('PMWorks_II', '0022_auto_20210527_1353'),
    ]

    operations = [
        migrations.CreateModel(
            name='WorkRequestFailureCause',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('Create', django_jalali.db.models.jDateTimeField(auto_now_add=True, verbose_name='تاریخ ایجاد')),
                ('Update', django_jalali.db.models.jDateTimeField(auto_now=True, verbose_name='تاریخ آخرین تغییر')),
                ('FailureCauseID', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='FailureCause', to='PMWorks_II.failurecause', verbose_name='علت خرابي')),
                ('WorkRequestID', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='WorkRequest', to='PMWorks_II.workrequest', verbose_name='درخواست کار')),
            ],
            options={
                'verbose_name': 'علت خرابي درخواست کار',
                'verbose_name_plural': 'علل خرابي دستور کار',
                'ordering': ['-Create'],
            },
        ),
    ]
